android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -2519,6 +2516,36 @@ public static int getNavigationBarHeight(Context context, KeyboardType keyboardT
     return (resourceId > 0) ? context.getResources().getDimensionPixelSize(resourceId) : 0;
   }
 
+  public static int getKeyboardHeightMax(Context context) {
```

Comment:
Can we include comments for these new public API functions?

Similar info to your Markdown help docs.